mappings/net/minecraft/block/InfestedBlock.mapping

```diff
@@ -1,11 +1,17 @@
 CLASS net/minecraft/class_2384 net/minecraft/block/InfestedBlock
 	FIELD field_11178 regularBlock Lnet/minecraft/class_2248;
 	FIELD field_11179 REGULAR_TO_INFESTED Ljava/util/Map;
-	METHOD <init> (Lnet/minecraft/class_2248;Lnet/minecraft/class_4970$class_2251;)V
+	METHOD <init> <init> (Lnet/minecraft/class_2248;Lnet/minecraft/class_4970$class_2251;)V
+		COMMENT Creates an infested block
```

Comment:
We really need to slove this engima issue

Reply:
I assume its the `<init> <init>` thing?